_sass/pages/_provider-map.scss

```diff
@@ -0,0 +1,52 @@
+$map-height-sm: 460px;
+$map-height-md: 560px;
+
+.provider-data-download-button {
+  width: 230px;
+}
+
+.provider-california-map {
```

Comment:
Speaking of the annoying link underline from CA State Template...

There is a hidden link control coming from Mapbox, but we still get the underline effect:

![image](https://user-images.githubusercontent.com/1783439/120836546-8a987180-c51a-11eb-8cca-bfca16512a17.png)

I think something like this would solve it:

```css
.provider-california-map {
    .mapboxgl-ctrl-bottom-left {
        display: none;
    }
}
```